pages/cat/[catId].tsx

```diff
@@ -109,20 +109,34 @@ const ShowCatById: React.FC<IndividualCatProps> = ({ cat, catImage }) => {
   );
 };
 
-export const getStaticPaths: GetStaticPaths = async () => {
-  const allCatsQuery = queryAllCats(1000, 0);
-  const { cats } = await request(ENDPOINT_MOONCAT_PROD, allCatsQuery);
-  const catIds = (cats as Cat[]).map((cat) => ({
-    params: { catId: cat.id },
-  }));
+// export const getStaticPaths: GetStaticPaths = async () => {
+//   const allCatsQuery = queryAllCats(1000, 0);
+//   const { cats } = await request(ENDPOINT_MOONCAT_PROD, allCatsQuery);
```

Comment:
do we want to remove the commented code here?

Reply:
Yeah of course. I kept there just as a fallback if getServerSide aint working .. will do that :)